### PR TITLE
Wapiti binaries for Windows x64

### PR DESCRIPTION
**Pull request by boumenot**
Wapiti rebuilt for Windows, and targeting x64.  These bits were built against JDK 1.7.

Most, but not all of the tests now execute.  There are at least two exceptions that I am aware of.

GROBID calls bash to execute pdftoxml, which is not available on Win32.
The test case *testGrobidModelsEnum_CustomModel* assumes UNIX paths.

### CODE REVIEW

File: grobid-core/src/main/java/org/grobid/core/main/LibraryLoader.java [hunk 1]
```diff
@@ -27,89 +27,92 @@
     private static boolean isContextMocked = false;
 
     public static void load() {
-        if (!loaded) {
```
Comment (boumenot) [inline grobid-core/src/main/java/org/grobid/core/main/LibraryLoader.java]: I inverted the if statement to reduce nesting, but the diff on GitHub is a mess.

Comment (boumenot) [inline grobid-core/src/main/java/org/grobid/core/main/LibraryLoader.java]: Nevermind, I combined the two dlls into one, and this change is not necessary.

**Comment by lfoppiano** [comment]
Hi @boumenot, 
     I've merged the change in the grobid master (I know I should have used a branch, I was too quick, but should not be a problem for the moment because the rest is untouched) and started testing. I'm working on a virtualized windows 7 64bit with 4Gb of Ram, Java 8. 

I've added the pdf2xml library for windows 64 (in fact I've used the 32bit to be quick, because I wanted have it working e2e). Since there were some library missing, I've downloaded them from ftp://ftp.zlatkovic.com/libxml/, now seems to work fine. Maybe you can have a look (it's in the grobid-home/pdf2xml/ subfolder). 

I've started testing using the PDFs and the conversion now works, the model is loaded but then I have an error calling wapiti and the application stops, see the log file: 

`[INFO] Starting scanner at interval of 10 seconds.
[DEBUG] org.grobid.service.process.GrobidRestProcessFiles: >> org.grobid.service.process.GrobidRestProcessFiles.methodLogIn
[DEBUG] org.grobid.core.utilities.IOUtilities: >> set origin document for stateless service'...
[DEBUG] org.grobid.core.factory.GrobidPoolingFactory: synchronized newPoolInstance
[INFO ] org.grobid.core.factory.GrobidPoolingFactory: Number of Engines in pool active/max: 1/10
[DEBUG] org.grobid.core.utilities.LanguageUtilities: synchronized getNewInstance
[DEBUG] org.grobid.core.analyzers.GrobidAnalyzer: Get new instance of GrobidAnalyzer
[INFO ] org.grobid.core.jni.WapitiModel: Loading model: C:\Users\chercheur\grobid-workspace\grobid\grobid-home\models\header\model.wapiti (size: 36094028)
[DEBUG] org.grobid.core.document.DocumentSource: start pdf2xml
[DEBUG] org.grobid.core.document.DocumentSource: Executing: [C:\Users\chercheur\grobid-workspace\grobid\grobid-home\pdf2xml\win-64\pdftoxml_server, -blocks, -noImage, -noImageInline, -fullFontName, -l, 2, C:\Users\chercheur\grobid-workspace\grobid\grobid-home\tmp\origin8320390077222524341.pdf, C:\Users\chercheur\grobid-workspace\grobid\grobid-home\tmp\P4mBP93nma.lxml]
[DEBUG] org.grobid.core.document.DocumentSource: pdf2xml process finished. Time to process:62ms
[Wapiti] Loading model: "C:\Users\chercheur\grobid-workspace\grobid\grobid-home\models\header\model.wapiti"
Model path: C:\Users\chercheur\grobid-workspace\grobid\grobid-home\models\header\model.wapiti
error: Cannot open input data buffer: arXiv arxiv a ar arX arXi v iv Xiv rXiv BLOCKSTART LINESTART NEWFONT HIGHERFONT 0 0 1 NOCAPS NODIGIT 0 0 0 0 0 0 0 0 0 0 NOPUNCT 0 0
: : : : : : : : : : BLOCKIN LINEIN SAMEFONT LOWERFONT 0 0 1 ALLCAP NODIGIT 1 0 0 0 0 0 0 0 0 0 PUNCT 0 0
nucl nucl n nu nuc nucl l cl ucl nucl BLOCKIN LINEIN SAMEFONT LOWERFONT 0 0 1 NOCAPS NODIGIT 0 0 0 0 0 0 0 0 0 0 NOPUNCT 0 0
- - - - - - - - - - BLOCKIN LINEIN SAMEFONT LOWERFONT 0 0 1 ALLCAP NODIGIT 1 0 0 0 0 0 0 0 0 1 HYPHEN 0 0

[.. cutted ..]

edu edu e ed edu edu u du edu edu BLOCKIN LINEEND SAMEFONT SAMEFONTSIZE 0 0 0 NOCAPS NODIGIT 0 0 0 0 0 0 0 0 0 0 NOPUNCT 0 0
1 1 1 1 1 1 1 1 1 1 BLOCKSTART LINESTART NEWFONT HIGHERFONT 0 0 0 NOCAPS ALLDIGIT 1 0 0 0 0 0 0 0 0 0 NOPUNCT 0 0


        <No error>

chercheur@chercheur-VBOX MINGW64 ~/grobid-workspace/grobid/grobid-service (master) $`

After that the application stops. This happens also with many of the tests that uses the model, but they don't fail (we need to fix this).

Have you ever had this kind of error before? Any idea where the problem could be?

**Comment by boumenot** [comment]
It has been about a year, but I vaguely remember working through these types of issues.  I'll still walking through my commits, and rebasing them on master.  

Let me complete that, and if that addresses these failures.

**Comment by boumenot** [comment]
I noticed one difference based on your message.  I built Wapiti against Java 7 not 8.  I originally built it against 8, but I kept getting unsupported version errors.  I noticed that the Travis builds against 7, so I switched from 8 to 7.

**Comment by lfoppiano** [comment]
No problem if it's not working at the first attempt. 
It's great anyway to have the binaries. Thanks to them I've managed to add all the building blocks around it, like pdf2xml and to fix all the hardcoded slashes in the tests and the code ;) 

Regarding Java, I've mentioned Java 8 because I'm using the JDK8 but I didn't change the compatibility version (it should be JDK7).

**Pull request merged**